boxoffice/static/js/models/util.js

```diff
@@ -19,25 +19,104 @@ export const Util = {
     var res = '₹' + otherNumbers.replace(/\B(?=(\d{2})+(?!\d))/g, ",") + lastThree + afterPoint;
     return res;
   }
-}
+};
 
-export const fetch = function(config){
+export const fetch = function (config) {
   return $.ajax({
     url: config.url,
     dataType: 'json'
   });
-}
+};
 
-export const post = function(config){
+export const post = function (config) {
   return $.ajax({
     url: config.url,
     type: 'POST',
-    dataType: 'json'
+    data: config.data,
+    contentType : config.contentType ? config.contentType : 'application/x-www-form-urlencoded; charset=UTF-8',
```

Comment:
Since this can either be a JSON object or a form object, it's better for config to accept a `json` flag. That way, the caller of `post` does not have to be aware of jQuery's `ajax` syntax, which should be an internal detail of `post`.